libnmstate/netinfo.py

```diff
@@ -92,3 +94,14 @@ def _ifaceinfo_bond(devinfo):
     if 'link-aggregation' in bondinfo:
         return bondinfo
     return {}
+
+
+def _remove_ovs_bridge_unsupported_entries(iface_info):
```

Comment:
Shouldn't the IP information be taken from the internal OVS interface instead? Is this PR meant to make internal interface visible in nmstatectl show output?

Reply:
At the moment, this abstraction of merging the bridge with the internal-interface is not applied.
We will have to do this in both direction (apply as well: A bridge implies also a port and an internal-interface of the same name).

For this PR, if you want an IP for the bridge, you need to explicitly define an internal-interface and attach it to the bridge.
Note that even if we'll implicitly create such an interface for a bridge, the ability to define new internal-interface is still needed.

Reply:
Please add an example to the examples directory (with appropriate tests) about how to use the internal interface, then, so we have at least documenation by example.